src/client.ts

```diff
@@ -160,6 +170,18 @@ export function sendVPythonCommandToServer(vPythonCommand: string) {
   }
 }
 
+// This function is used to send an updated state of the
+// enviro data server to the language server
+export function sendServerStateToServer(useServer: boolean) {
```

Comment:
Maybe change `sendServerStateToServer` -> `sendServerStateToLanguageServer` for clarity

Reply:
Fixed with 07ef593